include/swift/AST/Types.h

```diff
@@ -5473,6 +5480,11 @@ class ExistentialType final : public TypeBase {
   }
 };
 BEGIN_CAN_TYPE_WRAPPER(ExistentialType, Type)
+static CanExistentialType get(CanType constraint) {
+  return CanExistentialType(
+      ExistentialType::get(constraint, /*forceExistential*/ true)
```

Comment:
I think the `forceExistential` argument to `ExistentialType::get` was just a hack for type printing and I'm removing it in https://github.com/apple/swift/pull/41909. Why is it needed here?

Reply:
I want to guarantee an ExistentialType head for the force cast that ensues.

Reply:
I've refactored this code to assert this doesn't happen and dropped the bit. Thanks for the suggestion.